### PR TITLE
chore: add tox config for local testing

### PR DESCRIPTION
**Pull request by lili2311**
- [x] Ready for review
- [x] Follows CONTRIBUTING rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?

Current development documentation suggest dependencies are installed directly on the system without using a virtualenv, against a single python version. Both of these problems can be solved with `tox`.

#### Where should the reviewer start?

`tox.ini`

#### How should this be manually tested?

Clone a fresh copy of the repo. Run `pip install tox`. Running `tox` will now handle dependency install and test running.

This process should work in or out of a virtualenv.

#### Any background context you want to provide?

`tox` acts as a local replacement for the Travis CI build matrix, handling language and dependency installation.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+from setuptools import setup
+
+setup(
+    name="plug",
```
Comment (kyegupov) [inline setup.py]: what's plug needed for? a comment maybe?

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[tox]
+envlist = py27,py36
+
+[testenv]
+deps = -rdev-requirements.txt
```
Comment (miiila) [inline tox.ini]: would it work with `-r dev-requirements.txt` as well? Or is this tox specific?

File: setup.py (hunk 1 above)
Comment (lili2311) [reply setup.py]: plug is the python module in this codebase

File: tox.ini (hunk 2 above)
Comment (lili2311) [reply tox.ini]: the parser doesn't care about spaces, works with both

**Pull request merged**